space_missions.py

```diff
@@ -75,11 +75,13 @@ def load_mission_data(df):
         }
     
         postgres = space_etl.PostgresqlDestination(db_name=db_name)
-        schema_handle = space_etl.SchemaDriftHandle(db_name=db_name)
-        # Since we added columns in the transformation phase
-        
-        schema_handle.check_schema_drift(df=df,details=details)
-        
+        schema_handle = SchemaDriftHandle(db_name=db_name)
+        columns_to_add = schema_handle.check_schema_drift(df=df,details=details)
+        if columns_to_add:
+            for column_name in columns_to_add:
+                df_datatype = df[column_name].dtype
+                column_type = map_df_dtype_to_postgres(df_datatype)
+                schema_handle.add_columns(details=details,column_name=column_name,column_type=column_type)
 
         postgres.write_dataframe(df=df,details=details)
```

Comment:
This should not be here. The only function that should exist in ETL file is `write_dataframe`